prestodb/centos7-oj8/files/usr/local/bin/apply-site-xml-override

```diff
@@ -19,6 +19,6 @@ test -f "${site_xml}" || fail "${site_xml} does not exist or is not a file"
 test -f "${overrides}" || fail "${overrides} does not exist or is not a file"
 test ! -e "${site_xml_new}" || fail "${site_xml_new} already exists"
 
-xsltproc --param override-path "'${overrides}'" "/opt/prestodev/site-override.xslt" "${site_xml}" > "${site_xml_new}"
+xsltproc --param override-path "'${overrides}'" "/opt/prestodb/site-override.xslt" "${site_xml}" > "${site_xml_new}"
```

Comment:
This should be in the initial commit since the initial commit should build (no broken revisions)

Reply:
Should I merge them in to a single commit and force push?

Reply:
Yes please

Reply:
Done.